pa_matchbatch.sh

```diff
@@ -2,15 +2,7 @@
 set -e
 unset CDPATH
 
-# Arguments:
-# - REPO: Which repo is being released. `puppet-agent` or `puppet-agent-private`
-#         are the two primary options.
-# - WORKSPACE: Where the dependencies should be downloaded, e.g. `temp`.
-# - IGNORE_FOR: Which repos should be ignored (space-separated).
-# - OVERRIDE_PATH: A path to a txt file that will override versions.
-#
-# Deprecated arguments:
-# - PUPPET_AGENT_URL: This will still be read, but `REPO` is preferred.
+# See README.md for arguments and available environment variables.
```

Comment:
This was an incomplete list, so I replaced it with a reference to the README isntead of duplicating more